src/ciceroscm/thermal_factory.py

```diff
@@ -0,0 +1,26 @@
+from .upwelling_diffusion_model import UpwellingDiffusionModel
```

Comment:
If we keep expanding on this, would it make sense to have a common file for factory functions?